### PR TITLE
Fix broken coverage generation

### LINKED ISSUE #199: Istanbul generates errors with async/await
See https://github.com/gotwarlost/istanbul/issues/733

`yarn add istanbul@v1.1.0-alpha.1` fixes the problem.

### PR DESCRIPTION
**Pull request by neilbmclaughlin**
## Description

Stable version of istanbul doesn't handle async/await.
Upgrade to latest alpha version following discussion with the team.
Need to clear the coverage dir's since the file formats are not
backwards or forwards compatible between the two versions of istanbul.

## Related Issue

Fixes #199 

## Motivation and Context

Prevents warnings in the coverage testing

## Checklist
- [ ] Need to consider how we are going to move off the alpha version of the package.

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-201.dev.beta.nhschoices.net)

File: package.json [hunk 1]
```diff
@@ -5,10 +5,11 @@
   "main": "server.js",
   "scripts": {
     "build-css": "./css-build",
+    "coverage-clear": "rm -Rf coverage",
```
Comment (beseven) [inline package.json]: How does this affect the CI as it will be run there as well and the `coverage` directory is not checked in?

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-201.dev.beta.nhschoices.net)

File: package.json (hunk 1 above)
Comment (beseven) [inline package.json]: After re-running it on Travis, it seems fine

Comment (neilbmclaughlin) [inline package.json]: Since it's a clean file system `coverage` won't exist and the command will do nothing and exit with a 0.

**Pull request merged**